cinema/urls.py

```diff
@@ -13,9 +14,12 @@
 router.register("genres", GenreViewSet)
 router.register("actors", ActorViewSet)
 router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
+router.register("movies", MovieViewSet, basename="movie")
 router.register("movie_sessions", MovieSessionViewSet)
+router.register("orders", OrderViewSet)
```

Comment:
Code Style: Consider adding a blank line at the end of the file to adhere to the code style guidelines.